ReactWindows/ReactNative/Views/Progress/ReactProgressRingManager.cs

```diff
@@ -0,0 +1,83 @@
+﻿using ReactNative.UIManager;
+using ReactNative.UIManager.Annotations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace ReactNative.Views.Progress
+{
+    /// <summary>
+    /// Represents a native <see cref="ProgressRing"/> component.
+    /// </summary>
+    class ReactProgressRingManager : BaseViewManager<ProgressRing, ReactProgressRingShadowNode>
+    {
+        private const string ReactClass = "RCTProgressRing";
```

Comment:
RCT is kind of a legacy thing left over from iOS.  We reused it in some places for cross platform views (e.g., RCTView, RCTText, etc.), but other than that we haven't needed to use it.  Maybe just call it "WindowsProgressRing"?